astropy/table/pprint.py

```diff
@@ -13,19 +13,26 @@
 import numpy as np
 
 from .. import log
-# Note, in numpy <= 1.6, some classes do not properly represent themselves.
 from ..utils.console import Getch, color_print, terminal_size, conf
 from ..utils.data_info import dtype_info_name
 
-if six.PY3:
+__all__ = []
```

Comment:
Is it? In my past experience, not having it equals getting a bunch of crap when doing "from module import *". But this has nothing to do with the fix, so I'm okay with removing it if desired.